src/utils/ignore.ts

```diff
@@ -28,7 +28,7 @@ function validExtensions(context: ChildContext | RuleContext) {
 export function getFileExtensions(settings: PluginSettings) {
   // start with explicit JS-parsed extensions
   const exts = new Set<FileExtension>(
-    settings['import-x/extensions'] || ['.js'],
+    settings['import-x/extensions'] || ['.js', '.mjs', '.cjs'],
```

Comment:
Do we need `jsx` here?

Reply:
I don't think so, the `react` specific config adds the .jsx extension.  This is just defining the default / fallback extensions supported, when not using a config or using a config that doesn't define its own.